frontend/src/static/css/v_authorship.scss

```diff
@@ -31,6 +31,20 @@
         display: flex;
         margin-left: 3rem;
         margin-top: 1.0rem;
+
+        .checkboxes {
+          label {
+            align-items: center;
+            border-radius: 5px;
+            display: inline-flex;
```

Comment:
Seems like there's already a label selector below that applies the `display` and `align-items`, could remove these perhaps